### ISSUE TITLE
Maximum call stack size exceeded - babel-traverse

### ISSUE THREAD

**Issue by ligletg** [babel/babel#4973]
I'm getting a "Maximum call stack size exceeded" on each source file while starting my project with the following command :
```js
webpack-dev-server --config webpack.config.js --progress --colors --inline
```
### Log from webpack

```js
ERROR in ./src/form/ComboBox.jsx
Module build failed: RangeError: /Users/gliglet/dev/my_project/src/form/ComboBox.jsx: Maximum call stack size exceeded
    at NodePath.popContext (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/path/context.js:224:32)
    at TraversalContext.visitQueue (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:177:13)
    at TraversalContext.visitSingle (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:108:19)
    at TraversalContext.visit (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:192:19)
    at Function.traverse.node (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/index.js:114:17)
    at NodePath.visit (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/path/context.js:115:19)
    at TraversalContext.visitQueue (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitSingle (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:108:19)
    at TraversalContext.visit (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:192:19)
    at Function.traverse.node (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/index.js:114:17)
    at NodePath.visit (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/path/context.js:115:19)
    at TraversalContext.visitQueue (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitSingle (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:108:19)
    at TraversalContext.visit (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/context.js:192:19)
    at Function.traverse.node (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/index.js:114:17)
    at NodePath.visit (/Users/gliglet/dev/my_project/node_modules/babel-traverse/lib/path/context.js:115:19)
 @ ./src/index.js 172:17-43
```

### Environment
| software         | version
| ---------------- | -------
| Babel            |  6.20.0
| node             | 5.10.1
| npm              | 3.8.3
| Operating System | MacOSX 10.11.6

**Comment by DanialK** [comment]
We have a very similar issue, we haven't updated babel and our package.json is the same as yesterday but noticed this after fresh installing the packages this morning !

**Comment by tanx** [comment]
Same here.

**Comment by cbosco** [comment]
I've found downgrading babel-plugin-transform-object-rest-spread to `6.19.0` fixes this

**Comment by hzoo** [comment]
Check https://github.com/babel/babel/issues/4972 😄

**Comment by hzoo** [comment]
Just published a new version which should fix it (6.20.1)

**Comment by hzoo** [comment]
Going to close this in favor of https://github.com/babel/babel/issues/4972 - working on fix
